config/config.go

```diff
@@ -592,6 +617,9 @@ func New(v *viper.Viper) (*Configuration, error) {
 		c.BlacklistedAppMap[c.BlacklistedApps[i]] = true
 	}
 
+	if len(c.BlacklistedAccts) > 0 && c.BlacklistedAccts[0] != "" {
+		glog.Warningf(`Instead of blacklisted_accts, consider using account setting "disabled: true" or global setting "account_required: true".`)
```

Comment:
I will remove the warning. 
If the host will have defined accounts, it will be obvious they can be disabled individually.
Otherwise, not much point in starting with the blacklisted ones.